casa/Exceptions/Error2.cc

```diff
@@ -96,11 +96,11 @@ AipsError::~AipsError() noexcept
   // stack trace from last exception
   static String lastStackTrace = "*no-stack-trace*";
   // protects the lastMessage and lastStackTrace statics
-  static Mutex  lastErrorMutex;
+  static std::mutex  lastErrorMutex;
```

Comment:
One should probably also add an "#include <mutex>" to Error2.cc to make sure std::mutex is available.

Reply:
Good point. I've opened PR #1128 to add an "include mutex" if we want to ensure it is included in this file.
I was curious where mutex was being included from. It turns out  this file uses mutexes only when USE_STACKTRACE (inside ifdefs) and in that case there is an include of <casacore/casa/System/AipsrcValue.h> which includes mutex.